src/controllers/ownerMessageController.js

```diff
@@ -40,7 +46,10 @@ const ownerMessageController = function (OwnerMessage) {
   };
 
   const deleteOwnerMessage = async function (req, res) {
-    if (req.body.requestor.role !== 'Owner') {
+    if (
+      req.body.requestor.role !== 'Owner' &&
+      !(await helper.hasPermission(req.body.requestor, 'editHeaderMessage'))
+    ) {
```

Comment:
```suggestion
    if (
      !(await helper.hasPermission(req.body.requestor, 'editHeaderMessage'))
    ) {
```
Same as other comment

Reply:
Thank you, Nathan! I will fix it